app/services/host_aspects.rb

```diff
@@ -0,0 +1,93 @@
+module HostAspects
+  class Configuration
+    SUBJECTS = [:configuration]
+
+    SUBJECTS.each do |subject|
+      define_method "register_#{subject}".to_sym do |aspect_name, aspect_model = nil, &block|
+        self.send('register', subject, aspect_name, aspect_model, &block)
```

Comment:
why send?
should be 
```ruby
register(subject, aspect_name, aspect_model, &block)
```

(register is private, thus this would work, self.register won't, because Ruby :stuck_out_tongue_closed_eyes: